wasm/treenode_util.cc

```diff
@@ -0,0 +1,54 @@
+#include "treenode_util.h"
+
+#include "treenode.pb.h"
```

Comment:
Add tests for subtree isomorphism.

Reply:
Consider using Boost for subtree isomorphism instead: https://www.boost.org/doc/libs/1_58_0/libs/graph/doc/vf2_sub_graph_iso.html